x-pack/plugins/ml/public/application/components/help_menu/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useMlKibana } from '../../contexts/kibana';
+
+interface HelpMenuProps {
+  docLink: string;
+}
+
+export const HelpMenu: FC<HelpMenuProps> = React.memo(({ docLink }) => {
+  const { chrome } = useMlKibana().services;
+
+  useEffect(() => {
+    chrome.setHelpExtension({
+      appName: i18n.translate('xpack.ml.chrome.help.appName', {
+        defaultMessage: 'Machine Learning',
+      }),
+      links: [
+        {
+          content: i18n.translate('xpack.ml.chrome.helpMenu.documentation', {
+            defaultMessage: 'Documentation',
+          }),
+          href: docLink,
+          iconType: 'documents',
+          linkType: 'custom',
+          target: '_blank',
```

Comment:
Despite setting target to `_blank`, the link opens in the same tab for me. Same thing happens with the links for the Security solution I see. But for Dashboard, it opens in a different tab (which seems a better option). I can't work out why that it is, as the structure of the `<a>` tag looks identical.

Reply:
@lcawl Looks like the `linkType` here needs to be 'documentation'. That should fix the issue.